src/main/java/mil/dds/anet/database/OrganizationDao.java

```diff
@@ -95,6 +100,24 @@ public CompletableFuture<List<Organization>> getOrganizationsForPerson(
         personUuid);
   }
 
+  static class ResponsiblePositionsBatcher extends ForeignKeyBatcher<Position> {
+    private static final String sql =
+        "/* batch.getResponsiblePositionsForOrganization */ SELECT \"organizationResponsiblePositions\".\"organizationUuid\", "
+            + PositionDao.POSITIONS_FIELDS + " FROM \"organizationResponsiblePositions\" "
+            + "INNER JOIN positions on positions.uuid = \"organizationResponsiblePositions\".\"positionUuid\" "
+            + "AND \"organizationResponsiblePositions\".\"organizationUuid\" IN ( <foreignKeys> ) ";
```

Comment:
Here it becomes a matter of taste, whether JOIN's should use complex conditions or not. :smile: 
I'd use `WHERE` instead of the `AND` here.